orbit/cmd/desktop/desktop.go

```diff
@@ -302,6 +306,15 @@ func main() {
 					continue
 				}
 
+				// Check for null for backward compatibility with an old Fleet server
```

Comment:
Nit: While we're at it, let's update 305 with something like `log.Error().Err(err).Msg("get desktop summary")` (it used to be only policies initially)